packages/flutter/lib/src/material/dialog.dart

```diff
@@ -457,13 +457,18 @@ class _DialogRoute<T> extends PopupRoute<T> {
 
 /// Displays a dialog above the current contents of the app.
 ///
-/// This function typically receives a [Dialog] widget as its child argument.
-/// Content below the dialog is dimmed with a [ModalBarrier].
+/// This function typically receives a [WidgetBuilder] which build a [Dialog] widget
+/// as it's `builder` argument. Content below the dialog is dimmed with a [ModalBarrier].
```

Comment:
`its`

Though the way you phrased this it makes it sound like you mean the `Dialog`'s `builder` argument. :-)

Maybe "This function takes a `builder` which typically builds a [Dialog] widget.`

Reply:
Fixed!